### PR TITLE
Commonmark dependencies and test for html rendering

### PR DESCRIPTION
**Pull request by MC-Student**

### CODE REVIEW

File: src/main/java/commonmark/Renderer.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        renderer = HtmlRenderer.builder().build();
+    }
+
+    public String getHtmlRendering(String toRender)
+    {
+        Node document = parser.parse(toRender);
+        return renderer.render(document);
+    }
+
+    public boolean getHtmlFile(String rendered) throws IOException
```
Comment (andrewoid) [inline src/main/java/commonmark/Renderer.java]: Getters should return something.
This method writes out to a file which is unexpected given the name of the file.

File: src/main/java/commonmark/Renderer.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package commonmark;
```
Comment (andrewoid) [inline src/main/java/commonmark/Renderer.java]: Does it make sense to just have a commonmark package in our codebase?

File: src/main/java/commonmark/Renderer.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+import org.commonmark.node.*;
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
+import org.commonmark.Extension.*;
+import org.apache.commons.io.FileUtils;
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+
+public class Renderer
```
Comment (andrewoid) [inline src/main/java/commonmark/Renderer.java]: This is a pretty generic name.

File: src/main/java/commonmark/Renderer.java (hunk 2 above)
Comment (andrewoid) [reply src/main/java/commonmark/Renderer.java]: `touroll.markdown`

File: src/main/java/commonmark/Renderer.java (hunk 3 above)
Comment (MC-Student) [reply src/main/java/commonmark/Renderer.java]: MarkdownRenderer

File: README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ Java application to create video from multiple sources made by students at Touro
 
 [FFmpeg](https://ffmpeg.org/) - Generate video from images and audio\
 [ffmpeg4j](https://github.com/Manevolent/ffmpeg4j) - Java wrapper around FFmpeg\
-[commonmark-java](https://github.com/commonmark/commonmark-java) - Parse and Render Markdown\
+[touroll-java](https://github.com/commonmark/commonmark-java) - Parse and Render Markdown\
```
Comment (andrewoid) [inline README.md]: global find/replace is not your friend

File: src/main/java/touroll/markdown/MarkdownRenderer.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+import org.commonmark.parser.Parser;
+import org.commonmark.renderer.html.HtmlRenderer;
+import org.apache.commons.io.FileUtils;
+import java.io.File;
+import java.io.IOException;
+
+public class MarkdownRenderer
+{
+    Parser parser;
+    HtmlRenderer renderer;
```
Comment (andrewoid) [inline src/main/java/touroll/markdown/MarkdownRenderer.java]: private

File: src/main/java/touroll/markdown/MarkdownRenderer.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+        renderer = HtmlRenderer.builder().build();
+    }
+
+    public String getHtmlRendering(String toRender)
+    {
+        Node document = parser.parse(toRender);
+        return renderer.render(document);
+    }
+
+    public void wrHtmlStrToFile(String rendered) throws IOException
```
Comment (andrewoid) [inline src/main/java/touroll/markdown/MarkdownRenderer.java]: don't abbreviate things in Java.

**Review by andrewoid — APPROVED** [review]

**Pull request merged**